libethereum/ClientBase.cpp

```diff
@@ -527,3 +527,8 @@ int ClientBase::chainId() const
 {
 	return bc().chainParams().chainID;
 }
+
+std::tuple<h256, h256, h256> ClientBase::getWork()
```

Comment:
Can we remove it or do we need it? (some tests create `ClientBase`?)

Reply:
If it's because of `ClientTest`, better move this into `ClientTest`

Reply:
this is because of line 602 in eth.cpp
```
auto r = client()->getWork();
```

where client() is the interface

Reply:
Some class cannot be instantiated with `getWork()` being pure virtual. If this class is `ClientTest`, move this into `ClientTest`

Reply:
it is FixedClient

Reply:
can you move it into `FixedClient` then?